src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/integration/visits/VisitsMigrationIntTest.kt

```diff
@@ -138,6 +140,14 @@ class VisitsMigrationIntTest : SqsIntegrationTestBase() {
 
       verify(telemetryClient).trackEvent(eq("nomis-migration-visits-started"), any(), isNull())
       verify(telemetryClient, times(26)).trackEvent(eq("nomis-migration-visit-migrated"), any(), isNull())
+
+      await.atMost(Duration.ofSeconds(21)) untilAsserted {
```

Comment:
it should never take more than 21 seconds since we check every 10 seconds followed by 10 checks 1 second apart